setup.py

```diff
@@ -57,6 +57,7 @@ def readme():
     install_requires=[
         'python-dateutil',
         'thumbor>=6.0.0,<7',
+        'botocore==1.5.70',
```

Comment:
We can't fix the versions in here, we need to be compatible. What was the error you encountered?